### ISSUE TITLE
Discord Bot down?

### ISSUE THREAD

**Issue by Zuumbie** [JessicaYeh/rom-exchange-web#53]
ROMexchangeBot doesnt return any results

**Comment by JessicaYeh** [comment]
Yeah looks like it's down, thanks for the heads up! Trying to contact the bot hoster.

**Comment by JessicaYeh** [comment]
It is back up now!

**Comment by Zuumbie** [comment]
Thank you!

**Comment by Zuumbie** [comment]
Looks like it is down again for the past few days :) Thank you for your help! Also, is there anywhere I can donate for your hard work?

**Comment by JessicaYeh** [comment]
Hmm it looks like it's up on my end, and being actively used in the past few days. In your discord server does it show that the bot is online?

You can send donations to: https://paypal.me/JessicaYeh
Any amount is greatly appreciated! :)

**Comment by Zuumbie** [comment]
Yeah... its online.... I checked its permissions, nothing has changed. That's really weird. I'm not sure how to fix it.

**Comment by Zuumbie** [comment]
nevermind. figured it out. some of my admins tweaked the permissions and messed it up

**Comment by Zuumbie** [comment]
is it down again or is it our fault? :P

**Comment by JessicaYeh** [comment]
Yeah, it seems to be down :( Unfortunately I can't restart it myself. I need to contact the person hosting it to see if I can take over it.

**Comment by JessicaYeh** [comment]
@Zuumbie it's back up again! I'm hosting it now
